web/client/selectors/mapInfo.js

```diff
@@ -6,7 +6,7 @@
 * LICENSE file in the root directory of this source tree.
 */
 
-const { get, isArray } = require('lodash');
+const { get, omit, isArray } = require('lodash');
```

Comment:
they say that pluck is to be preferred to omit for performance reason.
(if you know precisely which props to pass. i mean)

Reply:
We should prefer readability over premature optimization